src/pybind/mgr/dashboard/frontend/src/app/core/error/error.ts

```diff
@@ -0,0 +1,33 @@
+export class DashboardError extends Error {
+  topHeader: string;
+  topMessage: string;
+  btmHeader: string;
+  btmMessage: string;
+  fa_icon: string;
+}
+
+export class DashboardNotFoundError extends DashboardError {
+  topHeader = $localize`Client Error`;
+  topMessage = $localize`Sorry, we have encountered a Client Error`;
+  btmHeader = $localize`Page Not Found`;
+  btmMessage = $localize`Sorry, we couldn’t find what you were looking for.
+  The page you requested may have been changed or moved.`;
+  fa_icon = 'fa fa-exclamation-circle';
```

Comment:
You could use this `Icons` helper (shared/enum/icons.enum.ts).

For example, in this case, it seems we're already using the exclamation triangle instead of the circle:
```suggestion
  icon = Icons.warning;
```
BTW, I'd remove the `fa_` from the input argument name as that (`font-awesome`) is an implementation detail, and classes should encapsulate that kind of details.